receiver/receiverhelper/obsreport.go

```diff
@@ -121,6 +122,24 @@ func (rec *ObsReport) EndMetricsOp(
 	rec.endOp(receiverCtx, format, numReceivedPoints, err, component.DataTypeMetrics)
 }
 
+// StartProfilesOp is called when a request is received from a client.
```

Comment:
I would also prefer to leave telemetry out at least until we figure out open-telemetry/opentelemetry.io/pull/5092 and we can mark this telemetry as unstable properly